lib/dkdeploy/php/helpers/http.rb

```diff
@@ -41,7 +41,7 @@ def http_get_with_redirect(url, limit = 5)
           http.read_timeout = fetch :http_read_timeout
           if url.scheme == 'https'
             http.use_ssl = true
-            http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+            http.verify_mode = OpenSSL::SSL::VERIFY_PEER
```

Comment:
`VERIFY_PEER` is the default and this line could actually be removed. But this way the change is more explicit